packages/terra-date-input/src/DateInput.module.scss

```diff
@@ -173,7 +173,7 @@
     }
 
     option:not([value='']) {
-      color: transparent;
+      color: #1c1f21;
```

Comment:
is this value provided by UX?

Reply:
Will this remain same for all the themes. Because base color is different for low-light theme : ` #b2b5b6 ` : https://github.com/cerner/terra-core/blob/c55faac00f280f8345ec0d4756530377b520446c/packages/terra-base/src/clinical-lowlight-theme/Base.scss#L13

Reply:
Yes, the color remains same because the color "#b2b5b6" is not clearly visible on white background color, for the low-light theme also we are having the white background color which is same as the default theme for the list options.

Reply:
how is it showing white here in UI site it looks different : 
<img width="1727" alt="Screenshot 2024-02-09 at 11 54 20 AM" src="https://github.com/cerner/terra-framework/assets/12869809/af79782c-d554-4336-94c4-3cae871cf62e">